app/views/invoice/pdf.haml

```diff
@@ -195,25 +195,20 @@
             %dt= t(:reg_no)
             %dd= @invoice.buyer_reg_no
 
-            - if @invoice.buyer_address.present?
-              %dt= t(:address)
-              %dd= @invoice.buyer_address
+            %dt= t(:address)
```

Comment:
I would remove custom translation key sitting at the root namespace (`address`), and use built-in `Invoice.human_attribute_name.buyer_address`. Or even better `Invoice::Buyer.address.human_attribute_name :address`.

Reply:
Went with Invoice::Buyer.address.human_attribute_name :address.